app/services/manual/update_service.rb

```diff
@@ -12,9 +12,12 @@ def call
 
     manual.draft
     manual.assign_attributes(attributes)
-    manual.save!(user)
-    reloaded_manual = Manual.find(manual.id, user)
-    Adapters.publishing.save_draft(reloaded_manual)
+
+    if manual.valid?
```

Comment:
This shouldn't really be necessary as `save!` should raise an exception if the model is not valid 🤔

Reply:
Having looked into this a bit further, we've agreed to stick with the current PR implementation, as this is how the `CreateService` solves the issue too. The `save!` method is not throwing an exception because it has been overridden. We don't really like this because the overridden method changes the contract provided by the default ActiveRecord `send!` method, but such is life without checked exceptions. If we were to change this it would need to be a separate PR that considered the whole application